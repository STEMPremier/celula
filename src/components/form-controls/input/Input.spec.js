/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

// eslint-disable-next-line import/named
import { Input } from '.';

describe('<Input />', () => {
  describe('render()', () => {
    test('render the Input component', () => {
      const wrapper = shallow(
        <Input htmlType="text" label="test label" name="test" />,
      );
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
