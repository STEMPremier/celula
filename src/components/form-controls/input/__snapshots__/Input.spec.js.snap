// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> render() render the Input component 1`] = `
<div
  className="ce-input ce-input--text"
>
  <label
    htmlFor="test"
  >
    test label
  </label>
  <div
    className="ce-input__container"
  >
    <input
      disabled={false}
      form=""
      icon=""
      id="test"
      max=""
      min=""
      name="test"
      onBlur={[Function]}
      onChange={[Function]}
      placeholder=""
      type="text"
      value=""
    />
  </div>
</div>
`;
